css/_variables.scss

```diff
@@ -4,7 +4,8 @@
  * Style variables
  */
 $baseFontFamily: -apple-system, BlinkMacSystemFont, 'open_sanslight', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$hangupColor: #bf2117;
+$hangupColor:#DD3849;
+$hangupHoverColor: #F25363;
```

Comment:
If I'm not mistaken, we should be careful to also add these to the branding repo.

Reply:
I'll keep this in mind. Thanks!